common/lib/capa/capa/templates/jsinput.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
 <section id="inputtype_${id}" class="jsinput"
     data="${gradefn}"
     % if saved_state:
```

Comment:
@nedbat There's some XML escaping happening in the lines below. Should that escaping be left as-is for JSInput?